roles/core/bluebanquise/tasks/main.yml

```diff
@@ -20,6 +20,7 @@
       skip: true
   tags:
     - internal
+    - always
```

Comment:
not detected

Reply:
why `always ` instead of `package ` tag ?

Reply:
The helper script gather_vars_checker.py did not detect the correct tags to use. I made a mistake in the script, I'm working on the fix at this exact moment :grin: .